### ISSUE TITLE
JSONParse error for 503 response in VA.gov service

### ISSUE THREAD

**Issue by ferristseng** [department-of-veterans-affairs/caseflow#11501]
https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/5148/events/405997/

We're getting a 503 response back from VA.gov when trying to validate an address, and they are returning an HTML response, which we are trying to parse as JSON.

```
<html><body><h1>503 Service Unavailable</h1>
No server is available to handle this request.
</body></html>
```

We may want to catch a 503 error and fail immediately before trying to parse it as JSON.

**Comment by ferristseng** [comment]
Happening for 504's as well

https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/5823/events/407238/

**Comment by lomaxap** [comment]
resolved with #11479
